authors/apps/authentication/tests/test_authentication.py

```diff
@@ -0,0 +1,124 @@
+from django.urls import reverse
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework .test import APITestCase, APIClient
+
+
+class RegisterAPITestCase(APITestCase):
+    """User manager tests"""
+
+    def setUp(self):
+
+        self.client = APIClient()
+
+        self.user_registration = {
```

Comment:
@3Nakajugo Please use fixtures so we can reuse the dummy data in our tests. You could have a look here `https://docs.djangoproject.com/en/2.2/topics/testing/tools/#topics-testing-fixtures`. Tell your colleagues follow a similar approach.